lib/u3d/commands.rb

```diff
@@ -110,6 +110,7 @@ def download(args: [], options: {})
 
         unity = check_unity_presence(version: version)
         return unless enforce_setup_coherence(packages, options, unity)
+        return if packages.empty?
```

Comment:
In any case, I fully agree that we need tests to cover every possible scenario available so we make sure that it is behaving precisely as we want to.